config/plugins/vimfiler.vim

```diff
@@ -57,6 +57,7 @@ function! s:vimfilerinit()
     silent! nunmap <buffer> gr
     silent! nunmap <buffer> gf
     silent! nunmap <buffer> -
+    silent! nunmap <buffer> s
```

Comment:
Because I use s as my windows func leader, so I need disable it in vimfiler buffer.